### PR TITLE
Docs: minor readability improvements

### PR DESCRIPTION
**Pull request by garymm**
IR.md:
None of these changes are meant to have any impact on the semantics of
the specification. They are just minor edits to make the text easier
to read.

Signed-off-by: Gary Miguel <garymiguel@microsoft.com>

### CODE REVIEW

File: docs/IR.md [hunk 1]
```diff
@@ -141,13 +131,13 @@ ir_version|int32|The ONNX version corresponding to the operators.
...
 domain|string|The domain of the operator set. Must be unique among all sets.
-opset_version|int64|The version of the set of operators.
-doc_string|string|A human-readable documentation for this set of operators. Markdown is allowed.
-operator|Operator[]|The operators of this operator set.
+opset_version|int64|The version of the operator set.
+doc_string|string|Human-readable documentation for this operator set. Markdown is allowed.
+operator|Operator[]|The operators contained in this operator set.
 
-The operator set version is a simple integer value that is monotonically increased as new versions of the operator set are published.
+The operator set version is a simple monotonically increasing integer.
```
Comment (gramalingam) [inline docs/IR.md:138]: The wording here is a bit confusing in the formal sense (though understandable). The earlier explanatory test seems to clarify the meaning of "increasing" better.

**Review by gramalingam — APPROVED** [review]

**Comment by gramalingam** [comment]
LGTM (with just one minor nit mentioned above).

File: README.md [hunk 2]
```diff
@@ -222,17 +222,17 @@ Check out the [contributor guide](https://github.com/onnx/onnx/blob/master/docs/
...
+[python_api]: docs/PythonAPIOverview.md
+[shape_inference]: docs/ShapeInference.md
+[version_converter]: docs/VersionConverter.md
+[new_op]: docs/AddNewOp.md
+[community]: community
+[sigs]: community/sigs.md
+[wgs]: community/working-groups.md
+[contributing]: docs/CONTRIBUTING.md
+[CMakeLists]: CMakeLists.txt
+[license]: LICENSE
```
Comment (jcwchen) [inline README.md:238]: https://github.com/onnx/onnx/pull/3376 ONNX recently updated these URLs as absolute links for PyPI description (https://pypi.org/project/onnx/) to access it as well

Comment (gramalingam) [reply README.md:238]: Thanks, good catch!

Comment (gramalingam) [reply README.md:238]: Is there some way of meeting both goals?

Comment (jcwchen) [reply README.md]: I think we might need more code to achieve (like string convert here https://github.com/onnx/onnx/blob/e3900651f056d559a72211c86f561f70d4c9f2e9/setup.py#L341), which would be quite complicated.

**Pull request merged**